CONTRIBUTING.adoc

```diff
@@ -87,6 +87,10 @@ In order to do builds locally, install the following build dependencies:
 ----
 $ sudo dnf install golang golang-race make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel
 ----
+The following might also me useful if you prefer as much of the build and test chain to be installed via rpm packages:
```

Comment:
You'd expect me to be able to write single sentence without typo. ;-)

Fixed, rebased on master -> 955653e0595aeb8b86c04cbd39a0076759b5db48.